exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = htmlLevel > 5;
+let cssLevelAbove5 = cssLevel > 5;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-let cssAndHtmlAbove5;
-let cssOrHtmlAbove5;
+let cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5 > 5;
```

Comment:
Although the script does produce a correct result, lines 20 and 21 are incorrect. I would suggest to check each variable you use (e.g. ``htmlLevelAbove5`` and ``cssLevelAbove5``) and what they actually are and see if the code makes sense